docs/tutorials/google/spin_echoes.ipynb

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
<p>But the constructed circuit above already contains only <code>sqrt(iSwaps)</code> and hence there's no need to convert it to the target gateset?</p><p> </p><p><code>MergeInteractionsToSqrtIswap</code> merges all consecutive (1 &amp; 2 qubit) interactions on two qubits into a unitary matrix and then decomposes this unitary using sqrt iswaps in an attempt to a) convert to the target gateset and b) reduce the circuit depth by reducing the number of operations.</p><p> </p><p>I suggest that we start with a circuit which contains 2q gates other than sqrt iswaps so we can actually demonstrate the basis conversion. We can write a small description explaining that the user is free to choose any one of the many available analytical decomposers in Cirq, each of which can have a specialized purpose. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Used a different two-qubit gate (<code>cirq_google.SYC</code>)  in this first example to show the effect of <code>MergeInteractionsToSwqrtIswap</code>.  Also added your nice description of what it does - thanks!</p>